test/utils/addUser.js

```diff
@@ -12,7 +12,19 @@ module.exports = async (user) => {
   const isValid = user && Object.keys(user).length !== 0 && user.constructor === Object;
   // Use the user data sent as arguments, else use data from fixtures
   user = isValid ? user : userData[0];
+  /*
+    Making a shallow copy of user allows us to safely modify the user object
+    without affecting the original fixture
+  */
+  user = { ...user };
+  const roles = user.roles;
+  delete user.roles;
```

Comment:
Please add a comment here of why we are doing this.

Reply:
Done